src/meta/meta_ckpt.c

```diff
@@ -864,14 +864,19 @@ __wt_ckpt_blkmod_to_meta(WT_SESSION_IMPL *session, WT_ITEM *buf, WT_CKPT *ckpt)
         if (FLD_ISSET(S2C(session)->timing_stress_flags, WT_TIMING_STRESS_BACKUP_RENAME) &&
           !F_ISSET(blk, WT_BLOCK_MODS_RENAME) && __wt_random(&session->rnd) % 10 == 0)
             skip_rename = true;
-
-        WT_RET(__wt_raw_to_hex(session, blk->bitstring.data, blk->bitstring.size, &bitstring));
-        WT_RET(__wt_buf_catfmt(session, buf,
-          "%s\"%s\"=(id=%" PRIu32 ",granularity=%" PRIu64 ",nbits=%" PRIu64 ",offset=%" PRIu64
-          "%s,blocks=%.*s)",
-          i == 0 ? "" : ",", blk->id_str, i, blk->granularity, blk->nbits, blk->offset,
-          skip_rename ? "" : F_ISSET(blk, WT_BLOCK_MODS_RENAME) ? ",rename=1" : ",rename=0",
-          (int)bitstring.size, (char *)bitstring.data));
+        if (!skip_rename && F_ISSET(blk, WT_BLOCK_MODS_RENAME)) {
+            WT_RET(__wt_raw_to_hex(session, blk->bitstring.data, blk->bitstring.size, &bitstring));
+            WT_RET(__wt_buf_catfmt(session, buf,
+            "%s\"%s\"=(id=%" PRIu32 ",granularity=%" PRIu64 ",nbits=0,offset=0,rename=1,blocks=)",
+            i == 0 ? "" : ",", blk->id_str, i, blk->granularity));
+        } else {
+            WT_RET(__wt_raw_to_hex(session, blk->bitstring.data, blk->bitstring.size, &bitstring));
+            WT_RET(__wt_buf_catfmt(session, buf,
+            "%s\"%s\"=(id=%" PRIu32 ",granularity=%" PRIu64 ",nbits=%" PRIu64 ",offset=%" PRIu64
+            ",rename=0,blocks=%.*s)",
+            i == 0 ? "" : ",", blk->id_str, i, blk->granularity, blk->nbits, blk->offset, 
+            (int)bitstring.size, (char *)bitstring.data));
```

Comment:
I think you don't need this change if you take the suggestion above. I'd prefer not to have two complex `wt_buf_catfmt` calls that are nearly identical.

Reply:
removed now!